src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -336,4 +352,64 @@ public String toString() {
           + " <topic-1>:<table-1>,<topic-2>:<table-2>,... ";
     }
   }
+
+  /* Validator to validate Kafka Provider values which says where kafka is hosted */
+  public static class KafkaProviderValidator implements ConfigDef.Validator {
+    public KafkaProviderValidator() {}
+
+    public void ensureValid(String name, Object value) {
```

Comment:
Instead of throwing can it return a boolean? Also where is this method used?

Reply:
This validator is used while defining the provider config and it is being invoked from the framework.

Reply:
Ahh, I should add override annotation as well..